aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -170,29 +170,38 @@ These concrete rules structure our communications.
 
 ## Capitalization
 
-The general rule is that things that are sentences should follow sentence capitalization rules. And, things that are titles/headings/labels should follow title capitalization rules.
-
-Sentences follow sentence capitalization rules:
-
+There are four types of capitalization. 
+1. Sentence case, where the first letter is a capital, the second and all subsequent words begin with lowercase letters.
+   (example: Storage volume) 
+2. Title case, where most words are capitalized except for "connector" words such as "the," "an," "or," etc.
+   (example: "The Benefits of Cloud Storage Explained")
+3. Camel case, where there are capital letters inside the word, almost always for branding purposes.
+    (example: GreenLake)
+4. All lower case, where every single word is lowercase 
+    (example: "lower case exudes an approachable and casual vibe")
+
+At HPE, almost all text should be written in sentence case. 
+
+Sentence case:
+- Attribute labels
+- Column headings
+- Legends
+- Tabs
+- Section and subsection headings
 - Paragraphs of text
 - Tips
 - Notifications
 - Help
 
-Titles follow title capitalization rules:
-
+Title case:
 - Page/dialog titles
-- Section headings
-- Attribute labels
-- Table column headings
-- Legends
-- Tabs/Buttons
+- Buttons
 
 Capitalize proper nouns. HPE brand and product names are proper nouns.
 
 Fight the urge to elevate normal nouns to proper noun status. For example, if you are in a "data sources" page and need to say something about data sources, do not capitalize it. Unless it is a true proper noun, leave it uncapitalized when following sentence capitalization rules.
 
-DO NOT USE ALL CAPS IN EITHER TITLES OR SENTENCES. It is harder to read and is equated with yelling.
+DO NOT USE ALL CAPS IN EITHER TITLES OR SENTENCES. It is equated with yelling and increases stress levels.
```

Comment:
Wondering if we shouldn't use all caps here

Reply:
@ericsoderberghp ? opinion?

Reply:
Agreed with Taylor here. I think we should always model the correct behavior unless clearly in a "Do not" section.